.travis.yml

```diff
@@ -31,7 +31,6 @@ matrix:
     fast_finish: true
     exclude:
         - php: 7.0
```

Comment:
Before we excluded the memcahed adapter on php 7.0. Now we exclude php 7.0.  😕 

You should remove the following as well: 

```yaml
exclude:
  - php: 7.0
```